src/fullnode.rs

```diff
@@ -103,6 +103,7 @@ pub struct Fullnode {
     broadcast_socket: UdpSocket,
     pub node_services: NodeServices,
     pub role_notifiers: (TvuRotationReceiver, TpuRotationReceiver),
+    role_senders: (TvuRotationSender, TpuRotationSender),
```

Comment:
@mvines, haalp. This turned a bit ugly. I need to cache the senders so that I can reliably abort when exit is signaled on the closure returned form run().